recipes/embree3/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+        if self.settings.compiler == "clang" and version < "4":
+            raise ConanInvalidConfiguration("Clang < 4 is not supported")
+        elif self.settings.compiler == "Visual Studio" and version < "15":
+            raise ConanInvalidConfiguration("Visual Studio < 15 is not supported")
+
+        if self._is_linux_clang_libcxx:
+            raise ConanInvalidConfiguration("conan recipe for Embree v{0} \
+                cannot be built with clang libc++, use libstdc++ instead".format(self.version))
+
+        elif self._is_mac_apple_clang_static:
+            raise ConanInvalidConfiguration("Embree cannot be built into static library with apple-clang,\
+                try shared library instead")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        target_name = "{0}-{1}".format("embree", self.version)
+        os.rename(target_name, self._source_folder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        # Configure CMake library build:
+        self._cmake.definitions["EMBREE_STATIC_LIB"] = not self.options.shared
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["EMBREE_TUTORIALS"] = False
+        self._cmake.definitions["EMBREE_GEOMETRY_CURVE"] = self.options.geometry_curve
+        self._cmake.definitions["EMBREE_GEOMETRY_GRID"] = self.options.geometry_grid
+        self._cmake.definitions["EMBREE_GEOMETRY_INSTANCE"] = self.options.geometry_instance
+        self._cmake.definitions["EMBREE_GEOMETRY_QUAD"] = self.options.geometry_quad
+        self._cmake.definitions["EMBREE_GEOMETRY_SUBDIVISION"] = self.options.geometry_subdivision
+        self._cmake.definitions["EMBREE_GEOMETRY_TRIANGLE"] = self.options.geometry_triangle
+        self._cmake.definitions["EMBREE_GEOMETRY_USER"] = self.options.geometry_user
+        self._cmake.definitions["EMBREE_RAY_PACKETS"] = self.options.ray_packets
+        self._cmake.definitions["EMBREE_RAY_MASK"] = self.options.ray_masking
+        self._cmake.definitions["EMBREE_BACKFACE_CULLING"] = self.options.backface_culling
+        self._cmake.definitions["EMBREE_IGNORE_INVALID_RAYS"] = self.options.ignore_invalid_rays
+        self._cmake.definitions["EMBREE_ISPC_SUPPORT"] = False
+        self._cmake.definitions["EMBREE_TASKING_SYSTEM"] = "INTERNAL"
+
+        self._cmake.configure(build_folder=self._build_folder)
+        return self._cmake
+
+    def build(self):
+        os.remove(os.path.join(self._source_folder, "common", "cmake", "FindTBB.cmake"))
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("LICENSE.txt", src=self._source_folder, dst="licenses")
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
```

Comment:
What is the config filename, and imported target if any?

Reply:
which config ?

Reply:
the CMake config file generated in `os.path.join(self.package_folder, 'lib', 'cmake')`

Reply:
Oh there are lots of them if I remember correctly. I can check it perhaps later.

Reply:
```
/lib/cmake/embree-3.12.0/embree-config.cmake
/lib/cmake/embree-3.12.0/embree-config-version.cmake
/lib/cmake/embree-3.12.0/sys-targets.cmake
/lib/cmake/embree-3.12.0/sys-targets-release.cmake
/lib/cmake/embree-3.12.0/math-targets.cmake
/lib/cmake/embree-3.12.0/math-targets-release.cmake
/lib/cmake/embree-3.12.0/simd-targets.cmake
/lib/cmake/embree-3.12.0/simd-targets-release.cmake
/lib/cmake/embree-3.12.0/lexers-targets.cmake
/lib/cmake/embree-3.12.0/lexers-targets-release.cmake
/lib/cmake/embree-3.12.0/tasking-targets.cmake
/lib/cmake/embree-3.12.0/tasking-targets-release.cmake
/lib/cmake/embree-3.12.0/embree_sse42-targets.cmake
/lib/cmake/embree-3.12.0/embree_sse42-targets-release.cmake
/lib/cmake/embree-3.12.0/embree_avx-targets.cmake
/lib/cmake/embree-3.12.0/embree_avx-targets-release.cmake
/lib/cmake/embree-3.12.0/embree_avx2-targets.cmake
/lib/cmake/embree-3.12.0/embree_avx2-targets-release.cmake
/lib/cmake/embree-3.12.0/embree_avx512skx-targets.cmake
/lib/cmake/embree-3.12.0/embree_avx512skx-targets-release.cmake
/lib/cmake/embree-3.12.0/embree-targets.cmake
/lib/cmake/embree-3.12.0/embree-targets-release.cmake
```

Reply:
so `self.cpp_info.names["cmake_find_package"]` and `self.cpp_info.names["cmake_find_package_multi"]` should be `embree`. What are the imported targets in `*-targets` files?